app/js/controllers.js

```diff
@@ -98,6 +102,9 @@ function RoomCtrl($scope, $routeParams, $timeout, socket) {
     $scope.placeholderVotes = voteArr;
 
 
+    var total =  _.reduce(_.map(_.pluck($scope.votes, 'vote'), parseFloat), sumOfTwo, 0);
```

Comment:
There's a bug here.  parseFloat will choke if anyone has voted ½ or with a non-numeric vote.  Non-numeric votes should be excluded and ½ translated to 0.5.

Reply:
This PR was open almost 7 years before it go a review (and another 1.5 years before I noticed). I'm not mad -- that's amazing!

Reply:
Amazing but accidental.  From my side, it was meant as a bug report through a non-standard channel than a code review.  I must have encountered the bug in the wild before I even started looking for the root cause.

Reply:
Sorry, that came across like a complaint and a I didn't mean for it to be. I just thought it was amusing -- pretty cool actually -- that you saw the PR _and responded to it_ after such a long time. And that it took me such a long time to see that you had responded. 

Open source is hard.

Reply:
Hard, and honestly amazing (often amusing, too).  No irony read or intended.